README.md

```diff
@@ -12,13 +12,22 @@ This is the swagger API [documentation](https://script-heroes-bn-backend-stgng.h
 
 This the heroku staging Api https://script-heroes-bn-backend-stgng.herokuapp.com/
 
-This the production Api   https://script-heroes-bn-backend.herokuapp.com/
----
+## This the production Api https://script-heroes-bn-backend.herokuapp.com/
 
+## Setup Dotenv
+
+- Create .env file in project root directory
+- Copy keys in .env.example file, which is in the project root directorym and assign values to those keys.
+- You can add more environmenta variables to the .env file
+- To use declared variables, require dotenv at the top of your file `require('dotenv').config()`
+- Access environment variable value by using `process.env.KEY_NAME` where KEY_NAME is the variable name.
+
+Note: If you make changes that uses environmental variables make sure to add those variables with example values in the .env.example file.
 
 ## Sequelize and Sequelize-cli
 
 # 1 Installing cli
+
```

Comment:
If you view this file [here](https://github.com/atlp-rwanda/script-heroes-bn-backend/blob/669e4649fd6bdff089931f9db52b92bb18db7369/README.md) , the content under the sequelize heading look a bit too large. Could you fix that.

Reply:
Yeah, I got it fixed.